components/algorithmsList/algorithmCard/style.module.css

```diff
@@ -24,6 +24,21 @@
   pointer-events: none;
 }
 
+:global(.dark) .title::after {
+  content: "";
+  position: absolute;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: 25px;
+  background: linear-gradient(
```

Comment:
You only need the background property, you can leave the rest out.

Reply:
Fixed. Thanks.